spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -101,104 +111,74 @@ def block_real_instant_verify_requests
   end
 
   before do
+    allow(progressive_proofer).to receive(:threatmetrix_plugin).and_return(threatmetrix_plugin)
+    allow(threatmetrix_plugin).to receive(:proofer).and_return(threatmetrix_proofer)
+
     allow(progressive_proofer).to receive(:resolution_proofer).and_return(instant_verify_proofer)
     allow(progressive_proofer).to receive(:lexisnexis_ddp_proofer).and_return(threatmetrix_proofer)
     allow(progressive_proofer).to receive(:state_id_proofer).and_return(aamva_proofer)
 
     block_real_instant_verify_requests
   end
 
+  it 'assigns threatmetrix_plugin' do
+    expect(described_class.new.threatmetrix_plugin).to be_a(
+      Proofing::Resolution::Plugins::ThreatMetrixPlugin,
+    )
+  end
+
   describe '#proof' do
     before do
       allow(IdentityConfig.store).to receive(:proofer_mock_fallback).and_return(false)
     end
 
     subject(:proof) do
       progressive_proofer.proof(
-        applicant_pii: applicant_pii,
-        ipp_enrollment_in_progress: ipp_enrollment_in_progress,
-        request_ip: Faker::Internet.ip_v4_address,
-        threatmetrix_session_id: threatmetrix_session_id,
+        applicant_pii:,
+        ipp_enrollment_in_progress:,
+        request_ip:,
+        threatmetrix_session_id:,
         timer: JobHelpers::Timer.new,
-        user_email: Faker::Internet.email,
-        current_sp: current_sp,
+        user_email:,
+        current_sp:,
       )
     end
 
-    context 'remote proofing' do
-      it 'returns a ResultAdjudicator' do
-        expect(proof).to be_an_instance_of(Proofing::Resolution::ResultAdjudicator)
-        expect(proof.same_address_as_id).to eq(nil)
+    context 'remote unsupervised proofing' do
```

Comment:
This diff is a little messy, but my goal is to, at the end of this refactor, have two top level contexts here: `remote unsupervised proofing` and `in-person proofing`.